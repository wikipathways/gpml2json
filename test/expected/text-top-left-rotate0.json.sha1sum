b1ba95086023f53cf02883f54cfd0c172b452781 ?-
