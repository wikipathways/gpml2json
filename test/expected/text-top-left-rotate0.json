{"pathway":{"@context":["https://cdn.rawgit.com/wikipathways/WpVocabularies/7a46a05/contexts/pvjs.jsonld"],"backgroundColor":"white","borderWidth":0,"color":"black","contains":["f5840"],"drawAs":"rect","gpmlElementName":"Pathway","height":181.48936170212767,"kaavioType":"Group","name":"test case","padding":5,"type":["Pathway"],"width":232.5531914893617,"x":0,"y":0,"zIndex":0,"fontSize":12,"fontWeight":"bold","textAlign":"left","verticalAlign":"top","dataSourceVersion":"20171102","standardName":"test case","displayName":"test case","textContent":"Name: test case","isSimilarTo":"http://wikipathways.org/index.php/Special:SearchPathways?query=test%20case&doSearch=1","id":"hash://sha256/276b1a713ea8c5ca2f7e0f600c7f6957a05bf510ba5c91d33f5e0998db2103ef"},"entityMap":{"f5840":{"id":"f5840","textContent":"Label for\nthe rendering of\ntext","textAlign":"left","x":19.57446808510636,"y":47.872340425531746,"backgroundColor":"#ffffcc","fontSize":10,"height":103.61702127659592,"drawAs":"rect","borderRadius":15,"width":182.97872340425533,"zIndex":28672,"color":"#000000","textDecoration":"none","fontFamily":"Arial","fontStyle":"normal","fontWeight":"normal","borderWidth":1,"verticalAlign":"top","lineHeight":1.1,"overflow":"hidden","textOverflow":"clip","whiteSpace":"pre","padding":8,"type":["Label","SingleFreeNode"],"kaavioType":"SingleFreeNode","gpmlElementName":"Label"}}}
